src/app/core/services/phase.service.ts

```diff
@@ -90,6 +90,19 @@ export class PhaseService {
     this.repoChanged$.next(repo);
   }
 
+  async changeRepositoryIfValid(repo: Repo) {
```

Comment:
Let's add comments similar to the style in phase.service.ts

Reply:
Also make `changeCurrentRepository` private and remove comments to 
`If on Issue Dashboard, add previously visited repositories to otherRepos`
and 'provide repo suggestions' because the the implementation has been changed. 

This is so that it is clear to developers to use `changeRepositoryIfValid` to call `changeCurrentRepository`